packages/ui/src/elements/PublishButton/index.tsx

```diff
@@ -139,7 +139,7 @@ export const DefaultPublishButton: React.FC<{ label?: string }> = ({ label: labe
               if (isActive) {
                 return (
                   <PopupList.ButtonGroup key={locale.code}>
-                    <PopupList.Button onClick={() => publishSpecificLocale(locale.code)}>
+                    <PopupList.Button onClick={() => [publishSpecificLocale(locale.code), close()]}>
```

Comment:
The array is a bit strange, probably should be avoided. I'd expect it to be `onClick={() => { publishSpecificLocale(locale.code); close(); }}>` or similar.